geniza/templates/snippets/language_switcher.html

```diff
@@ -5,7 +5,6 @@
         <fieldset>
             <input name="next" type="hidden" value="{{ redirect_to }}">
             <select name="language">
-                {% get_current_language as LANGUAGE_CODE %}
```

Comment:
Right, this is set for us because we have `django.template.context_processors.i18n` enabled in our settings.  Good catch on removing it.